File: dynamic_listings/lua/default/computer_science/enums2.lua
-- Lua doesn't have native enums, but we can use tables to simulate them
local FUEL_TYPES = {
    GASOLINE = 1,
    DIESEL = 2,
    LPG = 3
}


if (car.fuel_type == FUEL_TYPES.GASOLINE) then
    gasolinepump:use();
elseif (car.fuel_type == FUEL_TYPES.DIESEL) then
    dieselpump:use();
elseif (car.fuel_type == FUEL_TYPES.LPG) then
    lpgpump:use();
end

File: dynamic_listings/python/default/computer_science/enums2.py
from enum import IntEnum


class FuelTypes(IntEnum):
    GASOLINE = 1
    DIESEL = 2
    LPG = 3


if car.fuel_type == FuelTypes.GASOLINE:
    gasolinepump.use()
elif car.fuel_type == FuelTypes.DIESEL:
    dieselpump.use()
elif car.fuel_type == FuelTypes.LPG:
    lpgpump.use()
